vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/CookieSteps.java

```diff
@@ -194,7 +194,6 @@ public void setAllCookies(ExamplesTable parameters)
         {
             cookieManager.addCookie(row.get("cookieName"), row.get("cookieValue"), row.get("path"), currentUrl);
         });
-        navigateActions.refresh();
```

Comment:
1. Why it is nor required?
2. If it still not, step description should be updated